pkg/scheduler/framework/plugins/volumerestrictions/volume_restrictions.go

```diff
@@ -158,6 +158,11 @@ func haveOverlap(a1, a2 []string) bool {
 	return false
 }
 
+// fast path if there is no conflict checking targets.
+func noneConflictVolume(v *v1.Volume) bool {
```

Comment:
em...... struct is enough I think,  no need to use pointer here.

Reply:
Updated, but out of curiosity, can you explain why?

Reply:
Go is a language with GC, so if the struct is not big one, prefer to use struct to avoid GC for the sake of performance.